src/jsx.d.ts

```diff
@@ -2333,6 +2333,27 @@ export namespace JSXInternal {
 		placeholder?: string | undefined | SignalLike<string | undefined>;
 		playsInline?: boolean | undefined | SignalLike<boolean | undefined>;
 		playsinline?: boolean | undefined | SignalLike<boolean | undefined>;
+		popover?:
+			| 'auto'
+			| 'hint'
+			| 'manual'
+			| boolean
+			| undefined
+			| SignalLike<'auto' | 'hint' | 'manual' | boolean | undefined>;
+		popovertarget?: string | undefined | SignalLike<string | undefined>;
+		popoverTarget?: string | undefined | SignalLike<string | undefined>;
+		popovertargetaction?:
+			| 'hide'
+			| 'show'
+			| 'toggle'
+			| undefined
+			| SignalLike<'hide' | 'show' | 'toggle' | undefined>;
+		popoverTargetAction?:
```

Comment:
This will set the property whereas popoverTarget will set the attribute due to the fact the property is actually popoverTargetElement (and takes an element reference rather than an idref string). 

This mismatch might be confusing to people?

Reply:
I've just remembered I think the property for action reflects to the content attribute so it might be fine?

Reply:
Yep, should actually be okay because of that.